runtime/configs/proto/ibc_config.proto

```diff
@@ -12,6 +12,15 @@ message IBCConfig {
     // 2. The node is a servicer and thus when IBC enabled is they are enabled
     //    to relay IBC packets using an IBC relayer binary
     bool enabled = 1;
-    string private_key = 2; // hex encoded
-    string stores_dir = 3;
+    IBCHostConfig host = 2;
+}
+
+message IBCHostConfig {
+    string private_key = 1; // hex encoded
+    string stores_dir = 2;
+    BulkStoreCacherConfig bulk_store_cacher = 3;
+}
+
+message BulkStoreCacherConfig {
+    uint64 max_height_stored = 1;      // maximum number of heights to be stored in the cache before pruning
```

Comment:
I left another comment around how this is implemented. You say this is the `maximum number of heights` but it is actually the ACTUAL height during the height check.

I think there's a descpenecy/issue/confusion in tbothe documentation & implementation.

1. Please update both
2. If there aren't any tests, please add them
3. If it is meant to be the `number`, make the comment say it's a "circular buffer" like object.

I'm going to put a bit more though into proper naming here.